astropy/coordinates/sky_coordinate.py

```diff
@@ -217,11 +217,15 @@ def __init__(self, *args, copy=True, **kwargs):
                 setattr(self, attr, kwargs[attr])
 
         coord_kwargs = {}
+        component_names = frame.representation_component_names
```

Comment:
We should mention `pm_ra_cosdec`, `pm_dec`, and `radial_velocity` in the docstring!

Reply:
done